lib/declaration.ts

```diff
@@ -1031,15 +1066,31 @@ export default class DeclarationTranspiler extends base.TranspilerBase {
           this.emit('== null');
         }
         this.emit(') {');
-        this.emit(`return promiseToFuture(tt.${base.ident(mergedDeclaration.name)}`);
+        this.emit('return promiseToFuture(tt.');
+        this.visitName(mergedDeclaration.name);
         this.visitParameters(ts.createNodeArray(suppliedParameters), {namesOnly: true});
         this.emit('); }\n');
       } else {
         // No parameters were omitted, no null checks are necessary for this call
-        this.emit(`return promiseToFuture(tt.${base.ident(mergedDeclaration.name)}`);
+        this.emit('return promiseToFuture(tt.');
+        this.visitName(mergedDeclaration.name);
         this.visitParameters(ts.createNodeArray(mergedDeclaration.parameters), {namesOnly: true});
         this.emit(');\n');
       }
     }
   }
+
+  private emitExtensionGetterBody(declaration: ts.PropertyDeclaration|ts.PropertySignature) {
+    this.emit('return promiseToFuture(tt.');
```

Comment:
you should make `tt` a constant and replace all usages with the constant so it is easy to tweak `tt` to something else in the future